debugging/book-library/script.js

```diff
@@ -34,15 +36,24 @@ function submit() {
     pages.value == null ||
     pages.value == ""
   ) {
-    alert("Please fill all fields!");
+    showError("Please fill all fields!");
     return false;
-  } else {
-    let book = new Book(title.value, title.value, pages.value, check.checked);
-    library.push(book);
+  }if (myLibrary.some(b => b.title === title.value && b.author === author.value)) {
```

Comment:
Please, apply proper code formatting (spacing) to your code.

Reply:
Thank you for pointing that out! I have updated the code and applied proper formatting to improve readability and maintainability.